### ISSUE TITLE
speed up masked model for Tensorflow2

### ISSUE THREAD

**Issue by zhixiongzh** [microsoft/nni#4650]
Hi,

does NNI support the speed up masked model after pruning using Tensorflow2? In the document I only see the Pytorch version.

**Comment by zheng-ningxin** [comment]
Hi thanks for the valuable feedback! Currently, the speedup tool only support `pytorch`.

**Comment by zhixiongzh** [comment]
Thanks for the feedback!
